gammapy/estimators/points/core.py

```diff
@@ -849,3 +865,98 @@ def resample_axis(self, axis_new):
             fluxpoints.append(fp_new)
 
         return self.__class__.from_stack(fluxpoints, axis=axis_new)
+
+
+class LightCurve(FluxPoints):
+    """Specialized 'FluxPoints' heir class for lightcurves"""
+
+    def __init__(
+        self, data, reference_model, meta=None, gti=None, filter_success_nan=True
+    ):
+        if "time" not in data["norm"].geom.axes_names:
+            raise ValueError("Trying to build a lightcurve without time axis!")
+        super().__init__(data, reference_model, meta, gti, filter_success_nan)
+
+    @classmethod
+    def read(cls, filename, **kwargs):
+        kwargs.setdefault("format", "lightcurve")
```

Comment:
One would need to add a metadata class, as for the flux class, or re-used this metadata class